.github/workflows/deploy-website.yaml

```
@@ -2,7 +2,7 @@ name: Update `error-prone.picnic.tech` website content
 on:
   pull_request:
   push:
-    branches: [ master, website ]
+    branches: [ website ]
```

IIUC, if we wish to run the `build` job still on master, it should be added to the branches here. This will not overwrite the website as the `deploy` job does not run on master.

```suggestion
    branches: [ master, website ]
```

---

Ah that would be nice actually, warns us whenever there is an error there. As long as it doesn't override the website for every new merge :).